gusto/transport_equation.py

```diff
@@ -58,8 +58,8 @@ def __init__(self, state, V, ibp="once"):
 
         # default solver options
         self.solver_parameters = {'ksp_type':'preonly',
-                                  'pc_type':'bjacobi',
-                                  'sub_pc_type': 'ilu'}
+                                  'pc_type':'lu',
+                                  'pc_factor_mat_solver_package': 'mumps'}
```

Comment:
Do you need this for accuracy? Because the operator is no longer cell local? Would it be better then to use an iterative method?